bin/install_windows_10_sdk.ps1

```diff
@@ -1,18 +1,41 @@
+param ( # parameters need to be defined at the top of the script
+  [switch]$DryRun = $false
+)
+
 # Stop script execution when a non-terminating error occurs
 $ErrorActionPreference = "Stop"
 
-Write-Host "--- :windows: Installing Windows 10 SDK and Visual Studio Build Tools"
+Write-Output "--- :windows: Installing Windows 10 SDK and Visual Studio Build Tools"
 
 $windowsSDKVersionFile = ".windows-10-sdk-version"
 if (-not (Test-Path $windowsSDKVersionFile)) {
   Write-Output "[!] No Windows 10 SDK version file found at $windowsSDKVersionFile."
   exit 1
 }
 
-$windows10SDKVersion = Get-Content $windowsSDKVersionFile
+$windows10SDKVersion = Get-Content $windowsSDKVersionFile `
+  | Where-Object { $_ -match '\S' }  # Remove empty lines
```

Comment:
Worth testing if this other (and arguably more idiomatic) implementation would work too:
```suggestion
$windows10SDKVersion = (Get-Content -TotalCount 1 $windowsSDKVersionFile).Trim()
```

Though tbh not even sure the `.Trim()` is strictly needed

Reply:
> Though tbh not even sure the `.Trim()` is strictly needed

Turned out to be, but to account for leading and trailing whitespaces, not newlines. See 48410dbc98907c58e8c86cc1280c999ae5b35a24 and 85f7bc24116592482d15a9f6ddd8fc0b43975267